Mathlib/Data/Set/Function.lean

```diff
@@ -947,6 +948,19 @@ theorem BijOn.subset_left {r : Set α} (hf : BijOn f s t) (hrs : r ⊆ s) :
     BijOn f r (f '' r) :=
   (hf.injOn.mono hrs).bijOn_image
 
+theorem BijOn.insert (h₁ : BijOn f s t) (h₂ : f a ∉ t) :
+    BijOn f (insert a s) (insert (f a) t) := by
```

Comment:
Is there an `Iff` version of this?

Reply:
The other direction also needs `a ∉ s`, if I'm not mistaken

Reply:
Worth having as well, I would say